src/components/menu/menu-item/menu-item.component.js

```diff
@@ -32,6 +33,11 @@ const MenuItem = ({
   onKeyDown,
   variant = "default",
   showDropdownArrow = true,
+  // eslint-disable-next-line no-unused-vars
+  keyboardOverride,
+  ariaLabel,
+  m = 0,
```

Comment:
These two are now in the prop table with no description FYI

Reply:
Moved these to only be defined in the render